python_ipware/python_ipware.py

```diff
@@ -236,12 +228,12 @@ class IpWare(IpWareMeta, IpWareProxy, IpWareIpAddress):
     def __init__(
         self,
         precedence: Optional[Tuple[str, ...]] = None,
-        leftmost: bool = True,
-        proxy_count: int = 0,
+        leftmost: Optional[bool] = True,
```

Comment:
I'm not sure why you make `leftmost` an `Optional[bool]`. Passing `leftmost=None` seems to have the same effect as `leftmost=False`.

Reply:
We want to stay boolean, True for default value, False for reverse.